pypit/artrace.py

```diff
@@ -698,6 +706,111 @@ def trace_object(slf, det, sciframe, varframe, crmask, trim=2,
     return tracedict
 
 
+def new_ignore_orders(edgdet, fracpix, lmin, lmax, rmin, rmax):
```

Comment:
Not a big deal, but a lot of PYPIT's functions are written in alphabetical order, for somewhat easier navigation. I can think of reasons why we should rethink this approach though (i.e. group certain functions together).

Reply:
I didn't really put any thought into the placement of these functions yet, but I can follow suit on this convention.